source/stylesheets/refills/_search-tools.scss

```diff
@@ -82,16 +82,21 @@
       @include position(absolute, 5px 5px auto auto);
       font-size: 0.7em;
     }
+  }
 
-    .hide-options {
-      margin-bottom: 0;
 
-      ol, button {
-        display: none;
-      }
+  .hide-options {
+    margin-bottom: 0;
+
+    ol, button {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line